<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>folderExistsDialog</class>
 <widget class="QDialog" name="folderExistsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The source and destination is identical</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_source">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Source</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_destination">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Destination</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_source_modified">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Modified</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_content_source_modified">
         <property name="text">
          <string notr="true">Today</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_source_folder_name">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Folder name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_content_source_folder_name">
         <property name="text">
          <string notr="true">folder</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_source_folder">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_content_source_folder">
         <property name="text">
          <string notr="true">/source/</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_destination_modified">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Modified</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_destination_folder_name">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Folder name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_content_destination_modified">
         <property name="text">
          <string notr="true">Today</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_content_destination_folder_name">
         <property name="text">
          <string notr="true">folder</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_destination_folder">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_content_destination_folder">
         <property name="text">
          <string notr="true">/destination/</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEditNewName">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SuggestNewName">
       <property name="text">
        <string>Suggest new &amp;name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBoxAlways">
       <property name="text">
        <string>&amp;Always perform this action</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Rename">
       <property name="text">
        <string>&amp;Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Merge">
       <property name="text">
        <string>Merge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Skip">
       <property name="text">
        <string>Skip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxAlways</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEditNewName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAlways</sender>
   <signal>clicked(bool)</signal>
   <receiver>SuggestNewName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
